bftengine/src/bftengine/ReplicaImp.cpp

```diff
@@ -2301,7 +2302,8 @@ void ReplicaImp::onMessage<CheckpointMsg>(CheckpointMsg *msg) {
                                                               msgIsStable,
                                                               msgState,
                                                               stateDigest,
-                                                              otherDigest));
+                                                              reservedPagesDigest,
+                                                              rvbDataDigest));
   LOG_INFO(GL, "My " << KVLOG(lastStableSeqNum, lastExecutedSeqNum, getSelfEpochNumber()));
```

Comment:
Good to remove the word "My" from the info log. We can use "Current" or leave that blank.